src/accelerate/optimizer.py

```diff
@@ -134,13 +134,22 @@ def eval(self):
         return self.optimizer.eval()
 
     def step(self, closure=None):
+        if is_lomo_available():
+            from lomo_optim import AdaLomo, Lomo
```

Comment:
Same thing as earlier, but let's move the detection for Lomo to happen in here rather than in the Accelerator. Then we can make an attribute `self.is_lomo_optimizer`